package.json

```diff
@@ -40,6 +40,7 @@
     "reflect-metadata": "^0.1.13"
   },
   "devDependencies": {
+    "@preact/preset-vite": "^2.5.0",
```

Comment:
Did you make these changes in the `/client` directory?

Reply:
I have pulled down the recent updates now, so this should be resolved and package.json shouldn't cause any conflcts